### DISCUSSION TITLE
Possible to define the data block size when receiving Bluetooth audio?

### DISCUSSION THREAD

**Discussion by poolster** [pschatzmann/ESP32-A2DP#513] [Q&A] [answered]
When receiving audio via Bluetooth, is it possible to force the software to send a predetermined blocksize to the **read_data_stream()** callback? I find in my application that I always receive 4096 byte blocks whereas my application runs more efficiently with smaller blocksize and it's too time critical too waste processing on chopping it up.

**Comment by pschatzmann** [comment, marked as answer]
No, I am not aware of any such functionality
